src/dg_022E0354.c

```diff
@@ -1,4 +1,4 @@
-#include "overlay_29_022E0354.h"
+#include "dg.h"
```

Comment:
Including the address in the file name feels a bit messy for files that already have non-placeholder names. Perhaps the address can be added as a comment instead.